From 1b2b0f1859204e5a2a021ad17ed7ceeab342d82b Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Sat, 2 Feb 2019 10:55:29 +0800
Subject: [PATCH 3/4] usb: dwc3: drop unused variable 'node'

To avoid below warning:
drivers/usb/dwc3/core.c:1154:27: warning: unused variable 'node'
[-Wunused-variable]
  struct device_node      *node = dev->of_node;
                             ^~~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/usb/dwc3/core.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/usb/dwc3/core.c b/drivers/usb/dwc3/core.c
index a0c606e..9c77e77 100644
--- a/drivers/usb/dwc3/core.c
+++ b/drivers/usb/dwc3/core.c
@@ -1151,7 +1151,6 @@ static void dwc3_core_exit_mode(struct dwc3 *dwc)
 static void dwc3_get_properties(struct dwc3 *dwc)
 {
 	struct device		*dev = dwc->dev;
-	struct device_node      *node = dev->of_node;
 	u8			lpm_nyet_threshold;
 	u8			tx_de_emphasis;
 	u8			hird_threshold;
-- 
1.7.9.5

